source/DomainTheory/Examples/Omega.lagda

```diff
@@ -25,13 +25,14 @@ module DomainTheory.Examples.Omega
 
 open PropositionalTruncation pt
 
+open import MLTT.Plus-Properties
 open import NotionsOfDecidability.Decidable
 
 open import UF.Equiv
 open import UF.EquivalenceExamples
 open import UF.ImageAndSurjection pt
 open import UF.Subsingletons-FunExt
-open import UF.SubtypeClassifier hiding (⊥)
+open import UF.SubtypeClassifier renaming (⊥ to 𝟘Ω ; ⊤ to 𝟙Ω)
```

Comment:
In other modules, I rename ⊥ to ⊥Ω and similarly for top. You may consider doing the same.